src/lib/viewport-uniforms.js

```diff
@@ -24,10 +24,15 @@ function calculateMatrixAndOffset({
 
   let projectionCenter;
   let modelViewProjectionMatrix;
+  let modelViewMatrixInv;
 
-  const modelViewMatrixInv = new Matrix4(viewMatrix);
+  const modelViewMatrix = new Matrix4(viewMatrix);
   const viewProjectionMatrix = new Matrix4(projectionMatrix).multiplyRight(viewMatrix);
 
+  if (modelMatrix) {
```

Comment:
I think this needs to be handled in the cases below. We really need a meter offsets example for one of the layers that use the new uniform.